src/routes/article-list.ts

```diff
@@ -20,7 +21,7 @@ export default (): AppMiddleware => (
       },
       [hydra.totalItems]: await articles.count(),
       [hydra.member]: {
-        '@list': [...articles],
+        '@list': [...await all(articles)],
```

Comment:
Micro-optimization (and doesn't make a difference given the current implementation is synchronous), but we should group promises into a single `await` when they're not ordered.